data/addons/pyramid_fullauth.yaml

```
@@ -0,0 +1,15 @@
+addon: true
+category:
+- authorization
+- authentication
+demoUrl: ''
+description: Pyramid fullauth’s provides full user registration and management
+  functionality for pyramid based web applications..
+docsUrl: https://pyramid-fullauth.readthedocs.io/en/latest/
+maintainers:
+- fizyk
+name: pyramid_fullauth
+projectUrl: ''
+pypiUrl: https://pypi.org/project/pyramid_fullauth
```

PyPI appends a trailing slash.

```suggestion
pypiUrl: https://pypi.org/project/pyramid_fullauth/
```